Contents/mods/Hydrocraft/media/scripts/Smithing.txt

```diff
@@ -791,7 +791,7 @@ recipe Unpack Pallet of Steel Pipes
 
 recipe Pack Pallet of Steel Pipes
     {
-    	HCSteelpipe=12,
+    	HCSteelpipe=48,
```

Comment:
not sure about this. ofc packing and unpacking should have the same amount, so it doesnt just delete stuff, but i assume that is just a mistake. but also getting lucky and finding a pallet in the world giving you 48 pipes is just soooo much. that is alot of steel.